xml/obs_package_formats.xml

```diff
@@ -282,4 +282,124 @@ systemctl enable sshd</screen>
   <title>&obs.ai;</title>
   <remark>toms 2017-08-18: TODO What is it and what is needed</remark>
  </sect1>
+
+ <sect1>
+  <title>&flatpak;</title>
+  <para>
+  The <link xlink:href="https://flatpak.org/">Flatpak</link> format can be used
+  to generate desktop apps for linux.
+  </para>
+  <para>
+  For building an installable flatpak bundle, you need a
+  <filename>flatpak.yaml</filename> manifest.
+  See
+  <link xlink:href="https://docs.flatpak.org/en/latest/manifests.html">Flatpak Manifests</link>
+  for the full documentation.
+  </para>
+  <para>
+  </para>
+  <itemizedlist>
+  <listitem><para>
+  Input format is <link xlink:href="https://yaml.org">YAML</link>. Although
+  flatpak also accepts JSON, we are using YAML in &obs;, because we have
+  a special additional field in form of a YAML comment.
+  </para></listitem>
+  <listitem><para>
+  You can use JSON as it is a subset of YAML. But be aware that flatpak
+  accepts non-standard <literal>//</literal> comments, while &obs; does not.
+  </para></listitem>
+  <listitem><para>
+  <literal>#!BuildVersion</literal> - Use this field to specify the version
+  of your app so the <literal>.flatpak</literal> file will be versioned.
+  Flatpak manifests do not have a version field.
+  </para></listitem>
+  <listitem><para>
+  The base images (freedesktop, gnome, kde) are maintained in the
```

Comment:
I would remove the parenthesis and use the project names. As an alternative, you can keep the parenthesis, but use a tag around the image names, for example, with `literal`: `<literal>kde</literal>`.

```suggestion
  The base images for Freedesktop, GNOME, or KDE are maintained in the
```